pkg/kubectl/describe.go

```diff
@@ -508,7 +507,7 @@ func describePod(pod *api.Pod, events *api.EventList) (string, error) {
 			}
 		}
 		describeVolumes(pod.Spec.Volumes, out, "")
-		fmt.Fprintf(out, "QoS Class:\t%s\n", qos.InternalGetPodQOS(pod))
+		fmt.Fprintf(out, "QoS Class:\t%s\n", pod.Status.QOSClass)
```

Comment:
i think i would like to somehow get a value set independent of the kubelet.

for example, i should be able to see the qos class of a pod that is pending scheduling.

also, do we have any validation to ensure that value does not change once set?